cs/di-defaultextensions.texy

```diff
@@ -0,0 +1,88 @@
+DI: Základní rozšíření pro kontejner
+************************************
+
+Nette obsahuje několik předdefinovaných [Nette\DI\CompilerExtension |api:] pro DI kontejner.
+
+ExtensionsExtension
+===================
+
+Díky tomuto rozšíření můžeme zaregistrovat další rozšiření pomocí konfiguračních NEON souborů.
+
+/---neon
+extensions:
+	dibi: Dibi\Bridges\Nette\DibiExtension22
+
+dibi:
+	host: localhost
+\---
+
+InjectExtension
+===============
+
+Pokud u služeb nastavíme příznak `inject: true`, aktivujeme tím předávání závislostí přes veřejné
+proměnné s anotací [@inject |di-usage#toc-inject-annotations] a metody [inject* | di-usage#toc-passing-by-an-inject-method].
+
+/---neon
+services:
+	blog.articles:
+		class: App\Model\Articles
+		inject: true
+\---
+
+/---php
+namespace App\Model;
+
+use Nette\Database\Connection;
+
+class Articles
+{
+
+	/** @var Connection @inject */
+	public $connection;
+
+}
+\---
+
+V základním nastavení je toto rozšíření aktivováno pouze pro Presenter.
+
+DecoratorExtension
+==================
+
+Dekorátor je velmi mocné rozšíření, které se hodí především pro abstraktní a base třídy, jako je
```

Comment:
Dekorátor je rozšíření, pomocí kterého můžeme nastavit tagy nebo doplnit setup o volání metody u všech služeb určité třídy nebo implementujícími určité rozhraní. 

Jako příklad nastavíme …